tests/rptest/redpanda_cloud_tests/high_throughput_test.py

```diff
@@ -380,6 +380,9 @@ def setup(self):
         self.redpanda.clean_cluster()
 
     def tearDown(self):
+        # Detect pod crashes beforew cleaning
+        self.redpanda.raise_on_pod_crash()
```

Comment:
Is there a self-test for this? 

I don't think this actually works because exceptions in tearDown are effectively ignored: the test will still pass.